versions/releases.py

```diff
@@ -195,6 +220,22 @@ def get_release_notes_for_version(version_pk):
     return response.content
 
 
+def get_release_notes_for_version(version_pk):
+    """Get the release notes content.
+
+    Tries S3 first, and fallback to old github release notes if not found in S3.
+    """
+    content = get_release_notes_for_version_s3(version_pk)
+    if content:
+        processed_content = convert_adoc_to_html(content)
+        content_type = "text/asciidoc"
+    else:
```

Comment:
Should we add a log message (either here or in the get_..._s3 function that we couldn't find the file there?

Reply:
`get_file_data` logs an error message if the file is not found.